src/cmd/domain.go

```diff
@@ -90,9 +91,14 @@ opslevel list domain
 			common.JsonPrint(json.MarshalIndent(list, "", "    "))
 		} else if isCsvOutput() {
 			w := csv.NewWriter(os.Stdout)
-			w.Write([]string{"NAME", "ID", "ALIASES"})
+			if err := w.Write([]string{"NAME", "ID", "ALIASES"}); err != nil {
+				panic(err)
+			}
 			for _, item := range list {
-				w.Write([]string{item.Name, string(item.Id), strings.Join(item.Aliases, "/")})
+				err := w.Write([]string{item.Name, string(item.Id), strings.Join(item.Aliases, "/")})
+				if err != nil {
```

Comment:
I'm not sure we actually care about the error here.  Why not just continue letting it write more lines?   Unfortunately logging errors here will screw up the output most likely too.  I think this is an OK error to skip over.

Reply:
☝️ keeping this in mind for a follow up PR. Really I'm inclined to fully build the string then do the actual write operation once. Also